calibration/DOFSTK/CGINS-DOFSTK-02978__20200804.csv

```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-2978,CC_frequency_offset,-8.309214e+002,source_file DOFST-K_SBE43F_SN_2978_Calibration_2020-08-04.zip > 2978.cal and SBE 43F O2978 SOCAdjusted 11Aug20.pdf
+43-2978,CC_oxygen_signal_slope,2.9978e-004,use adjusted Soc
```

Comment:
All values look good. Need to be sure this slope ('adjusted Soc') value is in the instrument.